src/containers/NotFound/index.jsx

```diff
@@ -0,0 +1,14 @@
+import { Link } from 'react-router-dom';
+import styles from './styles.module.css';
+
+const NotFound = () => (
+  <div className={styles.returnContent}>
+    <h1>404- NotFound. This is not a valid address</h1>
+    <p>Please go to the main page here:</p>
+    <Link to="/" className={styles.returnButton}>
```

Comment:
Let's get rid of this, the menu option to navigate to the home page is already visible in the header. Thoughts?